docker-compose.yml

```diff
@@ -19,45 +19,37 @@ x-env-mapping: &env
     - HISTSIZE=50000
     - HISTIGNORE=ls:exit:"cd .."
     - HISTCONTROL=erasedups
-    # Note: docker compose uses the values exported from .env for GID/UID if
-    # they exist. ./docker/fix_olympia_user.sh uses those variables to fix
-    # the uid/gid of the user to match the host if necessary.
-    - UID=${UID:-9500}
-    - GID=${UID:-9500}
+    # Note: docker compose uses the values exported from .env for HOST_UID if
+    # it exists. ./docker/entrypoint.sh uses this variable to fix
+    # the uid of the olympia user to match the host user id if necessary.
+    - HOST_UID=${HOST_UID:-}
```

Comment:
It is actually "somewhat" better IMO to use a unique name here for a few reasons:
- debugging the value set during docker build and runtime is easier, no collision
- understanding which is which is also easier
- it is clear when setting the .env file that you are tracking the host UID and not the olympia user id.